examples/bert/run_pretraining.py

```diff
@@ -291,6 +301,53 @@ def train_step(self, data):
         self.nsp_accuracy.update_state(data["next_sentence_labels"], nsp_preds)
         return {m.name: m.result() for m in self.metrics}
 
+class AdamLRSchedule(keras.optimizers.schedules.LearningRateSchedule):
+    """
+    Implements a Learning Rate Scheduler for Adam with Linear Warmup and
+    Linear Decay for a fixed number of training steps instead of epochs.
+    """
+    
+    def __init__(self,
+            learning_rate,
+            num_warmup_steps,
+            num_train_steps
+        ):
+        self.threshold_learning_rate = learning_rate
+        self.warmup_steps = num_warmup_steps
+        self.train_steps = num_train_steps
+        
+    
+    def __call__(self, step):
+        """
```

Comment:
I think we can probably just have one docstring for the class (and non for call).

Reply:
Alright, would you like me to use a single line comment for the `call` method or is it fine if it has none?